GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +68,428 @@ Mesh* AssimpImporter::getMesh(int index) {
 
     return mesh;
 }
+
+bool AssimpImporter::create_instance(JNIEnv *env, const char* class_name,
+        jobject& new_instance) {
```

Comment:
On Second thought, how about moving jassimp related code to different cpp files. IMHO to keep the GearVRf assimp code separate from jassimp, as we might be adding more functionality to it in future. I am not much familiar with jassimp, but I guess jassimp native code don't need GearVRf wrapper to assimp so easy to move to separate file. How about moving to contrib directory since its open source code.